scripts/lib/CIME/case_setup.py

```diff
@@ -210,7 +210,7 @@ def _case_setup_impl(case, caseroot, clean=False, test_mode=False, reset=False,
                 logger.info("Finished testcase.setup")
 
         # Some tests need namelists created here (ERP) - so do this if are in test mode
-        if test_mode:
+        if test_mode or get_model() == "acme":
```

Comment:
It's been a while since I fixed this. I think one of our models had buildnml stuff that depended on it being run at setup time.